devtools/module.mk

```diff
@@ -14,9 +14,9 @@ DEVTOOLS := \
 	devtools/md5table$(EXEEXT) \
 	devtools/make-scumm-fontdata$(EXEEXT)
 
-include $(srcdir)/devtools/*/module.mk
+include $(srcdir)/devtools/*/module.mk $(srcdir)/devtools/*/*/module.mk
```

Comment:
Please remove this change. Rather do the following:

add module.mk to devtools/create_bladerunner, which will contain include directive. Test it and then I'll merge